zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java

```diff
@@ -964,4 +1016,20 @@ private boolean parseBoolean(String key, String value) throws ConfigException {
                                       + ". Choose 'true' or 'false.'");
         }
     }
+
+    private int parseInt(String key, String value) throws ConfigException {
```

Comment:
static

Reply:
Done, but I am little confused why we need to change this method to static? Right now, this is not used in a static method. Maybe for future?